### PR TITLE
Fix favicon

### PR DESCRIPTION
**Pull request by MatthewJWalls**
## What does this change?

https://www.theguardian.com/2015-06-24-manifest.json currently results in a 500 and we get a lot exceptions coming into Kibana for it. When replicating this locally it looks like it's because it's trying to serve up a static asset from favicons-garnett, which doesn't exist, I'm assuming we deleted this after cleaning up from garnett.

The normal favicon assets look correct so I'm removing the if-statement branch code for when garnett is enabled.

@NataliaLKB - I just noticed this duplicates something your most recent cleanup pr, is that going out soon?

### CODE REVIEW

File: applications/app/templates/webAppManifest.scala.js [hunk 1]
```diff
@@ -2,70 +2,34 @@
...
-        "gcm"
-    ]
+    {
+        "type": "image/png",
+        "sizes": "114x114",
+        "src": "@{JavaScript(Static("images/favicons/roundel-114x114.png"))}"
     }
-
+],
+    "theme_color": "#005689",
```
Comment (NataliaLKB) [inline applications/app/templates/webAppManifest.scala.js]: We need to use the garnett enabled branch for colours: https://github.com/guardian/frontend/pull/19077/files#diff-04569251492d26f7bc9ab4febb769974L26

**Review by NataliaLKB — CHANGES_REQUESTED** [review]

**Comment by NataliaLKB** [comment]
Thanks for flagging and doing this work!

**Review by NataliaLKB — APPROVED** [review]
Awesome 💯

**Comment by SiAdcock** [comment]
My omission (#18980), thanks @MatthewJWalls

**Pull request merged**